File: ssh.py
import paramiko
ssh = paramiko.SSHClient()
ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
ssh.connect("27.64.30.83", username="thanh", password="1509", port=6767)
ssh.exec_command("cd ~/Desktop/SWT/testswt")
ssh.exec_command("git pull")

File: ssh.js
const fs = require('fs');
const path = require('path');
const {NodeSSH} = require('node-ssh');
const ssh = new NodeSSH();

ssh.connect({
    host: '27.64.30.83',
    username: 'thanh',
    port: 6767,
    password: "1509",
    tryKeyboard: true,
  })
.then(function() {
    ssh.execCommand("echo 123");
});